Product/product.html

```diff
@@ -0,0 +1,198 @@
+                <div class="badge">
+                    <i class="material-icons-outlined">shopping_cart</i>
+                    <div class="number dnd">0</div>
+                </div>
+            </section>
+        </nav>
+    </header>
+    <div class="wrapper">
+        <aside class="filter-container">
+            <div class="title">
+                <h2>Filters</h2>
+                <h2>Clear</h2>
+            </div>
+            <div class="price-filter">
+                <h2>Price</h2>
+                <div class="slider">
+                    <p>50</p>
+                    <p>150</p>
+                    <p>200</p>
+                </div>
+                <input type="range" name="price" id="">
+            </div>
+            <div class="category-filter">
+                <h2>Category</h2>
+                <div class="checkbox">
+                    <label for="Men Clothing">
+                        <input type="checkbox">
+                        Men Clothing
+                    </label>
+                </div>
+                <div class="checkbox">
+                    <label for="Men Clothing">
+                        <input type="checkbox">
+                        Men Clothing
+                    </label>
+                </div>
+            </div>
+            <div class="rating-filter">
+                <h2>Rating</h2>
+                <div class="radio">
+                    <label for="4start&above">
+                        <input name="rating" type="radio">
+                        4 Stars & above
+                    </label>
+                </div>
+                <div class="radio">
+                    <label for="3start&above">
+                        <input name="rating" type="radio">
+                        3 Stars & above
+                    </label>
+                </div>
+                <div class="radio">
+                    <label for="2start&above">
+                        <input name="rating" type="radio">
+                        2 Stars & above
+                    </label>
+                </div>
+                <div class="radio">
+                    <label for="1start&above">
+                        <input name="rating" type="radio">
+                        1 Stars & above
+                    </label>
+                </div>
+            </div>
+            <div class="sort-by-filter">
+                <h2>Sort by</h2>
+                <div class="radio">
+                    <label for="low-to-high">
+                        <input name="price-sort" type="radio">
+                        Price - Low to High
+                    </label>
+                </div>
+                <div class="radio">
+                    <label for="high-to-low">
+                        <input name="price-sort" type="radio">
+                        Price - High to Low
+                    </label>
+                </div>
+            </div>
+        </aside>
+        <main class="products-container">  
```

Comment:
The cards as shown in PRD are vertical , are we sure the horizontal cards will be ok? hope they are , do confirm with TAs!